README.md

```diff
@@ -2,11 +2,45 @@
 Allows for full control of the PicoBrew Pico S/C/Pro & Zymatic models.  Shout out to [@hotzenklotz](https://github.com/hotzenklotz/picobrew-server), Brian Moineau for PicoFerm API, @tmack8001 for Z series support & updates.  
 [Demo Server](http://ec2-3-136-112-93.us-east-2.compute.amazonaws.com/)
 
+## Supported Devices
+* Pico S/C/Pro: fully featured
+* Zymatic: fully featured
+* ZSeries (Beta, with additional development requirements for SSL support)
```

Comment:
Wouldn't mention that SSL requirement here as end-users might think this isn't build out yet.

Maybe flag it in the development setup section if running locally would require SSL termination first. This could be like in my local setup mitmproxy (to log requests from mothership in flight) or nginx (included in the raspberrypi image).

Reply:
Sounds good. Removed the SSL part.